data/cpu_bugs/autoyast/SLE-15-SP0/sles-15-kvm-guest-autoyast.xml

```diff
@@ -0,0 +1,259 @@
+        <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+        <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+        <import_gpg_key config:type="boolean">true</import_gpg_key>
+        <media_url>http://mirror.bej.suse.com/dist/install/SLP/SLE-15-Packages-LATEST/x86_64/dvd1/Module-Desktop-Applications/</media_url>
+        <product>sle-module-desktop-applications</product>
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+        <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+        <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+        <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+        <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+        <import_gpg_key config:type="boolean">true</import_gpg_key>
+        <media_url>http://mirror.bej.suse.com/dist/install/SLP/SLE-15-Packages-LATEST/x86_64/dvd1/Module-Legacy/</media_url>
+        <product>sle-module-legacy</product>
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+        <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+        <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+        <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+        <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+        <import_gpg_key config:type="boolean">true</import_gpg_key>
+        <media_url>http://mirror.bej.suse.com/dist/install/SLP/SLE-15-Packages-LATEST/x86_64/dvd1/Module-Server-Applications/</media_url>
+        <product>sle-module-server-applications</product>
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+        <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+        <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+        <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+        <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+        <import_gpg_key config:type="boolean">true</import_gpg_key>
+        <media_url>http://mirror.bej.suse.com/dist/install/SLP/SLE-15-Packages-LATEST/x86_64/dvd1/Module-Web-Scripting/</media_url>
```

Comment:
You are mixing NUE & BEJ repositories, this can cause problems or at least test slowness
Checkout tests with prepare_profile e.g. https://openqa.suse.de/tests/3056106
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/autoyast_qam/15_installtest.xml.ep
You could use something like `% if ($check_var->('OPENQA_HOSTNAME', 'openqa.suse.de') {` or BEJ URL so local repo will be used.

Reply:
OK, Let me check. Mixing two locations for speed up.
For OSD, only NUE is enough.
I will improve it.